File: GeeksForGeeks-Problems/CPP/String/Anagram.cpp
class Solution
{
public:
    bool areAnagrams(string &s1, string &s2)
    {
        if (s1.length() != s2.length())
            return false;
        sort(s1.begin(), s1.end());
        sort(s2.begin(), s2.end());
        for (int i = 0; i < s1.length(); i++)
        {
            if (s1[i] != s2[i])
            {
                return false;
            }
        }
        return true;
    }
};

File: GeeksForGeeks-Problems/JAVASCRIPT/Functional Programming/Anagram.js
/**
 * @param {string} a
 * @param {string} b
 * @returns {boolean}
 */
class Solution {
  areAnagrams(s1, s2) {
    // code here
    let temp1 = s1.split("").sort().join("");
    let temp2 = s2.split("").sort().join("");
    if (temp1 === temp2) {
      return true;
    }
    return false;
  }
}
